.gitignore

```diff
@@ -56,3 +56,7 @@ docs/_build/
 
 # PyBuilder
 target/
+
+# Virtual environments
+venv/
```

Comment:
For global gitignore you can do:

```
graingert@baleful:~$ cat ~/.gitconfig 
[user]
	email = tom@yplanapp.com
	name = Thomas Grainger
[push]
	default = simple
[core]
	excludesfile = /home/graingert/.gitignore_global
[rerere]
	enabled = true
graingert@baleful:~$ cat ~/.gitignore_global 
*.swp
*.swo
.venv/
venv/
```

also I generally use: `source .tox/py34/bin/activate`